modules/libcom/src/misc/epicsString.c

```diff
@@ -358,3 +360,79 @@ unsigned int epicsMemHash(const char *str, size_t length, unsigned int seed)
     }
     return hash;
 }
+
+static
+char lower(char c)
```

Comment:
We use `toupper()` in `epicsStrCaseCmp()` and `tolower()` in other libCom routines, not sure why you felt the need to define your own version.

Reply:
Because I forgot about the `to*` and only tried `man lower`...

Reply:
Okay. We normally cast the argument to int to prevent a compiler warning on VxWorks.

Reply:
Done.